profile/README.md

```diff
@@ -0,0 +1,43 @@
+<p align="center">
+<img src="https://electricracing.northeastern.edu/wp-content/uploads/2023/02/cropped-NER-logo-white.png" alt="drawing" width="200"/>
+</p>
+
+<p align="center">
+<a href="https://electricracing.northeastern.edu">Website </a> | <a href="https://finishlinebyner.com">FinishLine </a> | <a href="https://www.linkedin.com/company/northeasternelectricracing"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/LinkedIn.svg" width="16" height="16"> </a>
+| <a href="https://www.instagram.com/nuelectricracing/"> <img src="https://raw.githubusercontent.com/tandpfun/skill-icons/main/icons/Instagram.svg" width="16" height="16"> </a> | <a href="https://nerdocs.atlassian.net/wiki/spaces/NER/overview?homepageId=65735"> <img src="https://cdn.worldvectorlogo.com/logos/confluence-1.svg" width="16" height="16"> </a>
```

Comment:
it looks like there's an extra space in these two that are being underlined, you can do tags with no content without a closing tag by just doing something like <a href="bla" src="bla" />